helmfile/values/opendistro-es.yaml.gotmpl

```diff
@@ -163,12 +111,39 @@ elasticsearch:
                       openid_connect_url: http://dex.dex.svc.cluster.local:32000/.well-known/openid-configuration
                       openid_connect_idp.verify_hostnames: false
                       openid_connect_idp.enable_ssl: false
-                      subject_key: name
-                      roles_key: groups
+                      subject_key: {{ .Values.elasticsearch.sso.subject_key }}
+                      roles_key: {{ .Values.elasticsearch.sso.roles_key }}
                   authentication_backend:
                     type: noop
         {{ end }}
 
+        internal_users.yml: |-
```

Comment:
It can be mounted from secrets, see https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/helm/opendistro-es/values.yaml#L166-L171, but then we need to create that secret somehow.

Reply:
I think it should be possible to set the value from a file using something like

```
releases:
- name: opendistro-es
  ...
  set:
  - name: ...config.data.internal_users.yml
    file: "{{ requiredEnv "CK8S_CONFIG_PATH" }}/files/elasticsearch/internal_users.yml"
```

Not sure which I prefer though, but it would shorten the config files pretty much :thinking:

Reply:
Yes, we can totally do this but as I am sort of pressed for time is it alright to do this in a future issue?

Reply:
Yeah I agree with Olle, it would be great but let's get this nice config update in first!

Reply:
Absolutely, I guess both my questions should address future issues :slightly_smiling_face: